book/Zapp-Pipes/12.-Datasource-for-rokuTV.md

```diff
@@ -0,0 +1,82 @@
+# Data Source Provided (DSP) for Roku-TV
+
+**This documentation is in beta**
+
+The Roku system cannot parse the javascript code, so the typical DSP plugin don't work for the roku TV.
+
+In order to solve this issue, we need 2 steeps :
+
+1) Modify the `plugin_manifest.json` on the DSP
+
+2) Create an external rest service for provide the data received from the client API
+
+### Prerequisites
+
+Before the next step, please make sure you are familiar with the DataSourceProvider concept, documentation [here](Home.md) .
```

Comment:
Change the sentence to: Please make sure that you are familiar with the DataSource provider concepts which can be found here [here](Home.md)